geospaas/base_viewer/forms.py

```diff
@@ -1,74 +1,43 @@
 from django.contrib.gis import forms
 from django.db.models import Q
+from django.forms import CharField
 from django.utils import timezone
 from leaflet.forms.widgets import LeafletWidget
 
-from geospaas.base_viewer.models import SearchModelForLeaflet
-from geospaas.catalog.models import Dataset as CatalogDataset
+from geospaas.catalog.models import Source as CatalogSource
```

Comment:
Since you don't define new Source in this module, you don't have to import it with another name.
```suggestion
from geospaas.catalog.models import Source
```